test/python/picongpu/quick/pypicongpu/species/constant/ionizationmodel/ionizationmodelimplementations.py

```diff
@@ -31,10 +31,8 @@ class Test_IonizationModelImplementations(unittest.TestCase):
     def setUp(self):
         electron = Species()
         electron.name = "e"
-        mass_constant = Mass()
-        mass_constant.mass_si = constants.m_e
-        charge_constant = Charge()
-        charge_constant.charge_si = constants.m_e
+        mass_constant = Mass(mass_si=constants.m_e)
+        charge_constant = Charge(charge_si=constants.m_e)
```

Comment:
This is physically wrong (and was wrong before):
```suggestion
        charge_constant = Charge(charge_si=constants.elementary_charge)
```

Reply:
In a refactoring, it is soooo important to not touch the functionality. I work hard for that and whenever I still happen to remove a bug, I silently sneak in another somewhere else to have bug number conservation. XD Anyhow, I hadn't taken the time to teach physics to my recorded vim command.